src/Microsoft.DotNet.Interactive.Mermaid.Tests/MermaidKernelTests.cs

```diff
@@ -0,0 +1,268 @@
+    A[Client] --> B[Load Balancer]
+    B --> C[Server1]
+    B --> D[Server2]
+");
+        
+    }
+
+    [Fact]
+    public async Task can_specify_background_color()
+    {
+        using var kernel = new CompositeKernel
+        {
+            new MermaidKernel()
+        };
+
+        KernelCommandResult result = await kernel.SubmitCodeAsync(@"
+#!mermaid --display-background-color red
+    graph TD
+    A[Client] --> B[Load Balancer]
+    B --> C[Server1]
+    B --> D[Server2]
+");
+
+        var events = result.KernelEvents.ToSubscribedList();
+
+        var formattedData = events
+            .OfType<DisplayedValueProduced>()
+            .Single()
+            .FormattedValues
+            .Single(fm => fm.MimeType == HtmlFormatter.MimeType)
+            .Value;
+
+        var htmlDoc = new HtmlDocument();
+        htmlDoc.LoadHtml(formattedData.FixedGuid());
+
+        var style = htmlDoc.DocumentNode
+                .SelectSingleNode("//div[contains(concat(' ',normalize-space(@class),' '),' mermaidMarkdownContainer ')]")
+                .Attributes["style"].Value;
+
+        style.Should().Be("background-color:red");
+    }
+
+    [Fact]
+    public async Task can_specify_display_dimensions()
+    {
+        using var kernel = new CompositeKernel
+        {
+            new MermaidKernel()
+        };
+
+        KernelCommandResult result = await kernel.SubmitCodeAsync(@"
+#!mermaid --display-width 200px --display-height 250px
+    graph TD
+    A[Client] --> B[Load Balancer]
+    B --> C[Server1]
+    B --> D[Server2]
+");
+
+        var events = result.KernelEvents.ToSubscribedList();
+
+        var formattedData = events
+            .OfType<DisplayedValueProduced>()
+            .Single()
+            .FormattedValues
+            .Single(fm => fm.MimeType == HtmlFormatter.MimeType)
+            .Value;
+
+        var htmlDoc = new HtmlDocument();
+        htmlDoc.LoadHtml(formattedData.FixedGuid());
+
+        var style = htmlDoc.DocumentNode
+            .SelectSingleNode("//div[@id='00000000000000000000000000000000']")
+            .Attributes["style"].Value;
+
+        style.Should().Be(" width:200px;  height:250px; ");
+
+    }
+
+    [Fact]
+    public async Task can_use_extension_methods_from_the_kernel_extension()
+    {
+        var kernel = new CompositeKernel
+        {
+            new CSharpKernel(),
+            new MermaidKernel()
+        };
+
+        await kernel.SendAsync(new SubmitCode($@"#r ""{typeof(MermaidKernel).Assembly.Location}""", "csharp"));
+
+        var result = await kernel.SendAsync(new SubmitCode(@"
+using System;
+
+typeof(List<string>).ToUmlClassDiagram().Display();
+", "csharp"));
+
+        var events = result.KernelEvents.ToSubscribedList();
+
+        var diagram = events
+            .OfType<DisplayedValueProduced>()
+            .Single()
+            .Value as UmlClassDiagram;
+
+        this.Assent(diagram!.ToMarkdown().ToString(), _configuration);
```

Comment:
Assent seems like overkill for this test. If it compiles, it's good, no?

Reply:
correct, also I added mermaid to api contract tets